feature-utils/poly-look/src/visualisations/charts/bubble-charts/bubbleCluster/bubbleCluster.js

```diff
@@ -1,5 +1,5 @@
 import * as d3 from "d3";
-
+import { L12n } from "@polypoly-eu/silly-i18n/src/l12n";
```

Comment:
I think since the last change you can just do 

```js
 import { L12n } from "@polypoly-eu/silly-i18n/src/l12n"; 
```

It should be included in the generated dist/index.js. I might be wrong, however, but that should be the way of using it, so please tell me if it's not.

Reply:
yep yep just that i was trying to bypass having the sillyi18n as a dep on the rollup, but still complains for it.
by the way, there is a similar import "@polypoly-eu/silly-i18n/src/l12n" in another file - should I correct it there as well?

Reply:
Right, I did the same elsewhere. Do it in this PR if you want, and thanks.